Description: Fix issues with the manpage generation script.
 * Add alleg5 suffix to manual page file names to prevent
   possible name clashes. 
Author: Tobias Hansen <tobias.han@gmx.de>

--- a/docs/scripts/make_man.c
+++ b/docs/scripts/make_man.c
@@ -6,7 +6,7 @@
 
 #define MAX_NAMES 64
 
-static const char *SECTION = "3";
+static const char *SECTION = "3alleg5";
 static const char *MANUAL = "Allegro reference manual";
 static dstr last_header;
 static dstr names[MAX_NAMES];
--- a/docs/Refman.cmake
+++ b/docs/Refman.cmake
@@ -293,7 +293,7 @@
 
             set(outputs)
             foreach(entry ${entries})
-                list(APPEND outputs ${MAN_DIR}/${entry}.3)
+                list(APPEND outputs ${MAN_DIR}/${entry}.3alleg5)
             endforeach(entry)
 
             add_custom_command(
